packages/jest-cli/src/cli/processArgs.js

```diff
@@ -221,8 +223,26 @@ function processArgs() {
       if (argv.testEnvData) {
         argv.testEnvData = JSON.parse(argv.testEnvData);
       }
+
+      const yargsSpecialOptions = ['$0', '_'];
+      const allowedOptions = Object.keys(options).reduce((acc, option) => {
+        return acc
```

Comment:
can we get rid of the unnecessary return statements in the arrow function and use implicit returns? :)